src/asyncValidation.js

```diff
@@ -31,7 +31,7 @@ module.exports = (joiSchema, customSchema, overrides = {}) => {
     const schema = Joi.object().keys(joiSchema);
     options.context.values = values;
 
-    let { error } = await Joi.validate(values, schema, options);
+    let { error } = await schema.validate(values, options);
```

Comment:
Validating values against a schema is now performed with a method on the schema.